README.md

```diff
@@ -62,30 +62,40 @@
 
```

Comment:
![image](https://github.com/katfogy/vet-clinic/assets/93039359/9399b580-9a3c-4a3f-85e0-320c96b0a05d)

- [ ] It would be nice if you fix all the links highlighted above to make your readme more professional and descriptive 👍 because right now these highlighted links are broken 😨 Kindly fix them or remove them if you don't have any dedication sections for these links 🤔